src/backend/joanie/tests/core/test_api_admin_users.py

```diff
@@ -211,3 +211,110 @@ def test_admin_api_user_delete(self):
             "The requested resource was not found on this server.",
             status_code=404,
         )
+
+    def test_admin_api_user_me_anonymous(self):
+        """
+        Anonymous users should not be able to get user information
+        """
+        factories.UserFactory(is_staff=True, is_superuser=True)
+        response = self.client.get("/api/v1.0/admin/users/me/")
+
+        self.assertEqual(response.status_code, 403)
```

Comment:
Actually it should return a 401? Do you understand why the permission does not work like on other endpoints?

Reply:
As discussed IRL, this subject is apparently a known problem without a clear solution. Let's keep it that way for the moment and see if it really is a problem for the frontend...